### DISCUSSION TITLE
Is there a way to see if a addition software is already installed?

### DISCUSSION THREAD

**Discussion by ghost** [PhoenicisOrg/phoenicis#2345] [Q&A]
So for example,can I see if I already installed dxvk for a prefix?

**Comment by plata** [comment]
see https://github.com/PhoenicisOrg/phoenicis/issues/1824

**Reply by ghost** [reply]
If I understand it right,then it doesn't work right now?

**Reply by plata** [reply]
No, not currently.

**Reply by ghost** [reply]
Will it be added in the future?

**Reply by plata** [reply]
Maybe. As described in the issue, it's not trivial at all. Additionally, the use case itself is questionable. The purpose of Phoenicis is that you install an app and it will just work. So you should not be required to fiddle around with prefixes manually at all.

**Reply by ghost** [reply]
I mean you will need dxvk for many programs,so this feature has a use case.

**Reply by plata** [reply]
Even if many programs need it, it should simply be contained in their install scripts.

**Reply by ghost** [reply]
I mean for programs that don't have a install script.

**Reply by plata** [reply]
Best way will be to simply create an install script.
